src/App.java

```diff
@@ -1,5 +1,203 @@
+import java.util.Scanner;
+
 public class App {
+    public Random myRandom;
+    // creates a connection to the other two classes, Random.java and
+    // InputOutput.java
+    private InputOutput io = new InputOutput();
+    public int numStudents = 0;
+    // numStudents is how many students are the
+    public int studentsPerTable = 0;
+    // studentsPerTable is later established when the user sets it as whatever they
+    // would like
+    private int option = 1;
+    // this is used for the user to select what option they would like.
+    private String filename = "";
+    private String option2 = "";
+    // This will be used to name the file the user would like to create
+    private Boolean fileError = true;
+    private Boolean initialFileError = true;
+    private String names = "";
+
+    /**
+     * Where the program begins!
+     */
+    public void start() {
+       
+        openingMenu();
+    }
+
+    // This is utilzied to read the file the user has set up. It is able to count
+    // lines in the text file
+    public Random readBlock(String filename) {
+        Random s = new Random();
+        try {
+            io.openFile(filename);
+            initialFileError = false;
+        } catch (Exception e) {
+            io.output(e.toString());
+            return s;
+        }
+
+        System.out.println("reading...");
+        while (io.fileHasNextLine()) {
+            // this sets the variable n as the next line of the text file
+            String n = io.getNextLine();
+            // sets the file lines into the random.java file
+            s.setBlock(n);
+            // raises the number of students everytime there is another line in the text
+            // file
+            numStudents++;
+        }
+        return s;
+
+    }
+
+    public void openingMenu() {
+        System.out.print("Hello, welcome to Random Seat Generator!");
+        System.out.println(" Please select which of the following options you would like to do:");
+        System.out.println("[1] Randomize Seating");
+        System.out.println("[2] Edit Class Names");
+        System.out.println("[3] Create New Class");
+        System.out.println("[4] Rename Class");
+        System.out.println("[5] Delete Class/File");
+        // alows for the program to read the lines the user types in when prompted
+        Scanner input = new Scanner(System.in);
+        option = input.nextInt();
+        // int specifies that the line will be an int. If the line is not an int, the

```

Comment:
You shouldn't use your own scanner, since you have the `InputOutput` object whose purpose is to abstract the input and output with the user. If you need a method to read an int rather than a strong, you should add that method to the InputOutput class and call it on the `io` object. That said for this example it might be easier to read a string from the io object and then convert it to an `int` here.